README.md

```diff
@@ -12,53 +12,72 @@ For running Osmose Frontend with Docker go into the docker directory. You can al
 Manual Installation
 -------------------
 
+### Dependencies
+
+For Ubuntu Server 16.04 and 18.04
+
+```
+apt install python2.7 python2.7-dev virtualenv gcc pkg-config libpng-dev libjpeg-dev libfreetype6-dev postgresql-server-dev-all libgeos-dev g++ python-shapely nodejs npm
```

Comment:
This week we just switch to python3.

Reply:
I've seen your commits, thank you for this switch ! I didn't realize that you were close to completion, congrats !

Should I update this PR to only use python 3 instructions then ?

Reply:
If you wish to.